src/ontology/cl-edit.owl

```diff
@@ -9021,7 +9022,7 @@ SubClassOf(obo:CL_0000646 obo:CL_0000036)
 
 # Class: obo:CL_0000647 (multinucleated giant cell)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:add") Annotation(oboInOwl:hasDbXref "GOC:tfm") Annotation(oboInOwl:hasDbXref "ISBN:0702022918") Annotation(oboInOwl:hasDbXref "ISBN:0702024783") obo:IAO_0000115 obo:CL_0000647 "A phagocytic cell formed by the fusion of macrophages, occurs in chronic inflammatory responses to persistent microorganism such as M.tuberculosis, component of granulomas. Sometimes used to refer to megakaryocytes.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:add") Annotation(oboInOwl:hasDbXref "GOC:tfm") Annotation(oboInOwl:hasDbXref "ISBN:0702022918") Annotation(oboInOwl:hasDbXref "ISBN:0702024783") Annotation(oboInOwl:hasDbXref "PMID:33348900") Annotation(oboInOwl:hasDbXref "Wikipedia:Giant_cell") obo:IAO_0000115 obo:CL_0000647 "A phagocytic syncytia cell formed by the fusion of macrophages, occurs in chronic inflammatory responses to persistent microorganism such as M.tuberculosis, component of granulomas. Sometimes used to refer to megakaryocytes.")
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0000647 "BTO:0003107")
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0000647 "FMA:83035")
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0000647 "MESH:D015726")
```

Comment:
In the dbxref provided PMID:33348900 there is the acronym MGC.

Reply:
Added it